tox-win.ini

```
@@ -0,0 +1,19 @@
+[tox]
+envlist = py34,py35,py36,py37,cover
+skip_missing_interpreters = True
+
+[testenv]
+usedevelop = True
+deps = pytest
```

Instead of relisting these dependencies here can we use something like:
```suggestion
deps = acme[dev]
       .[dev]
```

---

Indeed. Tox updated accordingly, and I take the occasion to simplify it a bit.